Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -391,9 +414,11 @@ void AdvancedWidget::AddDescriptions()
       "possible input for external editing software.<br><br><dolphin_emphasis>If unsure, leave "
       "this at \"Aspect Ratio Corrected Internal Resolution\".</dolphin_emphasis>");
 #if defined(HAVE_FFMPEG)
-  static const char TR_USE_FFV1_DESCRIPTION[] =
-      QT_TR_NOOP("Encodes frame dumps using the FFV1 codec.<br><br><dolphin_emphasis>If "
-                 "unsure, leave this unchecked.</dolphin_emphasis>");
+  static const char TR_CODEC_CHOICE_DESCRIPTION[] = QT_TR_NOOP(
+      "Selects codec for encoding frame dumps. <br>MPEG4: Lossy. Good compression. Can set "
```

Comment:
```suggestion
      "Selects codec for encoding frame dumps.
      <br><br><b>MPEG4</b>: Lossy. Good compression. Can set "
```
For format consistency with other popups.